source/_integrations/fronius.markdown

```diff
@@ -88,6 +98,58 @@ Recommended energy dashboard configuration for meter location in feed in path (`
 - `Battery systems` aren't supported directly. Use [Template](/integrations/template) to split and invert negative values of `Power battery` entity (from power_flow) for charging power (W) and positive values for discharging power (W). Then use [Riemann sum](/integrations/integration/) to integrate each into energy values (kWh).
 - For `Devices` use the Ohmpilots `Energy consumed` entity.
 
+### Consumption path
+
+Recommended energy dashboard configuration for meter location in consumption path (`Meter location: 1`):
+
+This configuration has been possible thanks to [@GianlucaCollot work](https://github.com/GianlucaCollot), honors and credit go to him as I have just reproduced his work and documented it here :)
+
+First you need to create the power entities in your `configuration.yaml`. Make sure to replace the name of the sensor, you may only need to update the IP on the end if it was auto-detected:
+
+```yaml
+template:
+  - sensor:
+    # Positive values are from grid
+    - name: "power_bought"
+      unit_of_measurement: W
+      state: >-
+        {{ max(states('sensor.power_grid_fronius_power_flow_0_XXX_XXX_XXX_XXX') | float, 0) }}
+      device_class: power
+
+    # Negative values are to grid
+    - name: "power_sold"
+      unit_of_measurement: W
+      state: >-
+        {{ max((0 - states('sensor.power_grid_fronius_power_flow_0_XXX_XXX_XXX_XXX') | float), 0) }}
+      device_class: power
+```
+
+then, you need to create the energy entities that get data from the power ones (also in your `configuration.yaml` file):
+
+```yaml
+sensor:
+  - platform: integration
+    source: sensor.power_sold
+    name: energy_sold
+    round: 2
+    
+  - platform: integration
+    source: sensor.power_bought
+    name: energy_bought
+    round: 2
+```
+
+Now, you only need to configure Energy dashboard's params using the UI (first to values use the newly created entities, other ones are just the same as in the _"feed in path"_ config):
+
+- For `Grid consumption` use the meters `energy_bough` entity.
```

Comment:
"Grid consumption" isn't code, and should thus not be code fenced (same applies to a couple of other values below).

Reply:
Ok, was just to give attention to the field names in the UI.

Reply:
> Not sure if I should edit it.

🤷 I was reviewing your changes; not the full document.